### PR TITLE
Add march tech day email

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Add email reminder about the March tech days.

### CODE REVIEW

File: SR2019/2019-02-25-mar-techday.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+to: Student Robotics 2019
+subject: March tech day reminder
+---
+
+Hi all,
+
+Just a reminder and some information on the tech day we're running on on Saturday 9th March.
```
Comment (antoinepetty) [inline SR2019/2019-02-25-mar-techday.md]: "on on"

File: SR2019/2019-02-25-mar-techday.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Tech Days are opportunities for you to get help if you're struggling with a technical issue, and is a controlled environment to make more progress on your robots. It’s also an opportunity to see how well you're doing in comparison to other teams. This is the final tech day before the competition, and so is the last chance to get some hands-on help from us blue shirts.
+
+We're running this tech day in 2 locations, London and Southampton. The day will run from 10am until 5pm, however feel free to come and go as you please. Please let us know if you're interested in attending. The London venue will only run of there is sufficient interest, and spaces are limited to six teams.
```
Comment (antoinepetty) [inline SR2019/2019-02-25-mar-techday.md]: I'm wondering if we need to be explicit that we would like a response for either venue? Although it is fairly clear already.

File: SR2019/2019-02-25-mar-techday.md (hunk 1 above)
Comment (RealOrangeOne) [reply SR2019/2019-02-25-mar-techday.md]: woops, looks like we send that in the last one too :man_facepalming:

File: SR2019/2019-02-25-mar-techday.md (hunk 2 above)
Comment (RealOrangeOne) [reply SR2019/2019-02-25-mar-techday.md]: I was thinking that as I was writing it too. "Please let us know if you'd like to attend, and which venue you wish to go to" felt a little verbose, and a little obvious i'd hope. Worst comes to worst we can infer it from location / kickstart preference

**Comment by RealOrangeOne** [comment]
This has now been shipped.

**Pull request merged**